pygmt/tests/test_datasets_mean_dynamic_topography.py

```diff
@@ -0,0 +1,53 @@
+"""
+Test basic functionality for loading Earth mean dynamic topography datasets.
+"""
+
+import numpy as np
+import numpy.testing as npt
+from pygmt.datasets import load_earth_mean_dynamic_topography
+
+
+def test_earth_mdt_01d():
+    """
+    Test some properties of the Earth mean dynamic topography 01d data.
+    """
+    data = load_earth_mean_dynamic_topography(resolution="01d")
+    assert data.name == "z"
+    assert data.attrs["description"] == "CNES Earth mean dynamic topography"
+    assert data.attrs["units"] == "meters"
+    assert data.attrs["horizontal_datum"] == "WGS84"
+    assert data.shape == (181, 361)
+    assert data.gmt.registration == 0
+    npt.assert_allclose(data.lat, np.arange(-90, 91, 1))
+    npt.assert_allclose(data.lon, np.arange(-180, 181, 1))
+    npt.assert_allclose(data.min(), -1.47, atol=0.01)
```

Comment:
Based on https://github.com/GenericMappingTools/gmtserver-admin/blob/master/recipes/earth_mdt.recipe, the data has a precision of 0.0001 m. So, we should set `atol=0.0001`

Reply:
Thanks! Updated in commit https://github.com/GenericMappingTools/pygmt/pull/3718/commits/338377c735a24b1b9c6940ba15c73597a528edfe.